src/common/pure/ConfirmationPendingContent/ConfirmationPendingContent.tsx

```diff
@@ -0,0 +1,67 @@
+import { Trans } from '@lingui/macro'
+import { CheckCircle, UserCheck } from 'react-feather'
+import { ReactNode } from 'react'
+import { Wrapper } from './styled'
+import { UpperSection } from './styled'
+import { CloseIconWrapper } from './styled'
+import { WalletIcon } from './styled'
+import { LowerSection } from './styled'
+import { StepsWrapper } from './styled'
+import { StepsIconWrapper } from './styled'
+
+interface ConfirmationPendingContentProps {
+  onDismiss: () => void
+  statusIcon: ReactNode
+  title: string | ReactNode
+  description: string | ReactNode
+  operationSubmittedMessage: string
+  operationLabel: string
+  walletNameLabel?: string
+  walletAddress?: string
+}
+
+export function ConfirmationPendingContent({
```

Comment:
Thank you!
You actually implemented 99% of https://github.com/cowprotocol/cowswap/issues/2430